tests/test_calendar_privacy.py

```diff
@@ -0,0 +1,54 @@
+from app.internal.calendar_privacy import can_show_calendar
+# after user system is merged: from app.internal.security.dependancies import CurrentUser
```

Comment:
I think you can remove it right now..

Reply:
This is actually as mentioned in preparation of the merging of the user system - it's sort of a todo upgrade after that system is merged